examples/basic/main.tf

```diff
@@ -42,10 +42,18 @@ module "ocean-aks-np-vng" {
   tags                                      = { "key1": "value1", "key2": "value2" }
   labels                                    = { "labelkey1": "labelvalue1","labelkey2": "labelvalue2"}
   taints                                    = [{"key":"key1","value":"value1", "effect" : "NoSchedule"}]
-  vmsizes_filters_min_vcpu                  = 2
-  vmsizes_filters_max_vcpu                  = 16
-  vmsizes_filters_min_memory_gib            = 10
-  vmsizes_filters_max_memory_gib            = 18
-  vmsizes_filters_series                    = ["D v3"]
-  vmsizes_filters_architectures             = ["X86_64"]
+  vmsizes_filters_min_vcpu                 = 4
+  vmsizes_filters_max_vcpu                 = 96
+  vmsizes_filters_min_memory_gib           = 14.0
+  vmsizes_filters_max_memory_gib           = 900.0
+  vmsizes_filters_series                   = ["NC","NV"]
+  vmsizes_filters_exclude_series           = ["HB","HC"]
+  vmsizes_filters_architectures            = ["X86_64", "INTEL64", "ARM64","AMD64"]
+  vmsizes_filters_accelerated_networking   = "Disabled"
+  vmsizes_filters_disk_performance         = "Standard"
+  vmsizes_filters_min_gpu                  = 4.0
+  vmsizes_filters_max_gpu                  = 8.0
+  vmsizes_filters_min_nics                 = 1
+  vmsizes_filters_vm_types                 = ["generalPurpose", "GPU"]
```

Comment:
use other values, as these are used in cluster

Reply:
done